src/products/degen-mode.md

```
@@ -0,0 +1,42 @@
+
+## Option Lab: Entering as a Seller🤑
+
+Degen Seller mode will display basic values like Strike Price, Expiration Date, and Quantity. 
+
+| Selections               | Selection Meanings                                                                                          |
+|------------------------|------------------------------------------------------------------------------------------------------------------------|
+| [**Position Sentiment**](/options-for-starters/calls-and-puts#long-call-option-bullish)           | Sell a <span style="color: #4a85ff">**Put**</span> if you are [**Bullish**](/options-for-starters/calls-and-puts#long-call-option-bullish) :ox::chart_with_upwards_trend: -- or -- Sell a <span style="color: #4a85ff">**Call**</span> if you are [**Bearish**](/options-for-starters/calls-and-puts#long-put-option-bearish) :bear: :chart_with_downwards_trend: |
+| [**Strike Price**](/options-for-starters/intro#terminology)       | Select the base price that you will compare your Position Sentiment to: Will the price be going <span style="color: #4a85ff">**above**</span> or <span style="color: #4a85ff">**below**</span> the strike price??.           |
+| [**Expiration Date**](/options-for-starters/intro#terminology)    | Select the date your option contract <span style="color: #4a85ff">**Expires**</span> by. The contract is over after this date |
+
+## Options Contract Multiplier:100::
```

markdownlint-cli2 (0.17.2)

Trailing punctuation in heading
Punctuation: '::'

(MD026, no-trailing-punctuation)